### DISCUSSION TITLE
Removal of old commands

### DISCUSSION THREAD

**Discussion by fluca1978** [pgagroal/pgagroal#380] [Ideas]
With https://github.com/agroal/pgagroal/pull/289 merged, now commands like `flush-all` have been replaced by a new command line parsing syntax like `flush all`. Other commands have been changed, like `config-set` that is now `conf set` and so on.

So far, `pgagroal-cli` and `pgagroal-admin` will emit a warning when the user types an old command, and this is documented.

Sooner or later, we should remove the old commands from the command line.

I propose to keep the old commands at least for two minor releases, i.e., to remove them on `1.8` in order to let the users to switch to the new command line.

**Comment by jesperpedersen** [comment]
I agree with this

**Comment by fluca1978** [comment]
I am wondering if does make sense to let the application tell the user when a command will be removed.
So far, we have something like: `config-set has been deprecated since version 1.6`.
We could improve this with: `config-set has been deprecated since version 1.6 and will be removed in version 1.8`.

So, when we add another command that deprecates an old one, we could already explain to the user when this will be removed.

**Reply by jesperpedersen** [reply]
Nah, because we may go to 2.0 if I can find time for it...

Lets keep as is
